davidjeyachandran/alianzaamor
# [#54] checkin

To test: 
- Add user to delivery in the future
- Log in as a user
- Click on `Deliveries` tab
- You should see future delivery and ability to `Confirm` or `Reject`
- Once confirmed or rejected user should be able to cancel the previous action by seeing and clicking either `Cancel confirmation and reject` or `Cancel rejection and confirm`.

String to translate:
- View: https://alianzadeamoraqp.org/admin/structure/views/view/user_deliveries?destination=/admin/structure/views
- Deliveries
- Confirm
- Reject
- Cancel confirmation and reject
- Cancel rejection and confirm
- We received your confirmation.
- We received your rejection.
